.github/workflows/build_wheels_linux_arm.yml

```diff
@@ -0,0 +1,167 @@
+name: Build PYPI wheels for opencv-python on Linux ARM
+
+on:
+  pull_request:
+    branches:
+      - master
+  release:
+    types: [published, edited]
+
+
+jobs:
+  build_arm:
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ARM64]
+        python-version: [3.6, 3.7, 3.8, 3.9]
+        platform: [x64]
+        with_contrib: [0, 1]
+        without_gui: [0, 1]
+        build_sdist: [0]
+
+    env:
+      ACTIONS_ALLOW_UNSECURE_COMMANDS: true
+      REPO_DIR: .
+      BUILD_COMMIT: master
+      PROJECT_SPEC: opencv-python
+      MB_PYTHON_VERSION: ${{ matrix.python-version }}
+      TRAVIS_PYTHON_VERSION: ${{ matrix.python-version }}
+      PLAT: aarch64
+      MB_ML_VER: 2014
+      NP_TEST_DEP: numpy
```

Comment:
probably, we should set specific version (some old one?)

Reply:
At this moment, python 3.6 supports `numpy <= 1.19.5`, and python 3.9 supports `numpy => 1.19.3`, but there is an issue with `numpy==1.19.5` for the build on ARM. I defined `numpy == 1.19.4` in the latest commit.

Reply:
Got it, thanks, I was not aware of this issue. This way we can go with python-specific numpy versions, but it's for the future.

Reply:
I am sorry, I did not link the issue, it is here: https://github.com/opencv/opencv-python/issues/485